src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java

```diff
@@ -193,6 +195,25 @@ public MonitoredTrip(OtpDispatcherResponse otpDispatcherResponse) throws URISynt
         initializeFromItineraryAndQueryParams();
     }
 
+    /**
+     * Checks that, for each query provided, an itinerary exists.
+     * @return a summary of the itinerary existence results for each day of the week
+     */
+    public boolean checkItineraryExistence(
+        boolean replaceItinerary,
+        Function<OtpRequest, OtpResponse> otpResponseProvider
+    ) throws URISyntaxException {
+        // Get queries to execute by date.
+        List<OtpRequest> queriesByDate = getItineraryExistenceQueries();
+        this.itineraryExistence = new ItineraryExistence(queriesByDate, this.itinerary, this.arriveBy, otpResponseProvider);
```

Comment:
Nit. `this.` are not needed.

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/240/commits/df3b4b62278435205497ff8c28f88e9169faf815.